src/fiatconnect/types.ts

```diff
@@ -22,3 +22,5 @@ export class FiatConnectTxError extends Error {
     this.error = error
   }
 }
+
+export type QuoteResponseQuoteField = QuoteResponse['quote']
```

Comment:
nit: you could just inline this type in `CachedTransferDetails`. Don't think we get much out of this exported type